modules/ROOT/pages/indexes/search-performance-indexes/using-indexes.adoc

```diff
@@ -243,6 +243,28 @@ Text indexes are only used for exact query matches. To perform approximate match
 
 For more information about the predicates supported by text indexes, see xref:indexes/search-performance-indexes/managing-indexes.adoc#text-indexes-supported-predicates[Create, show, and delete indexes -> Text indexes: supported predicates].
 
+[[text-index-dictionary-variables]]
+=== Text indexes and dictionary variables
+
+Text indexes are not used to solve predicates with dictionary variables, such as JSON.
```

Comment:
Not sure about "such as JSON" wording. You can use Cypher maps to represent JSON objects...